unanet-summarizer.js

```diff
@@ -24,10 +24,16 @@ window.summarizeUnanetTime = (function() {
         return new Function(code.replace(/[\r\t\n]/g, ''));
     };
 
+    const SUMMARIZER_ROOT = 'https://excellalabs.github.io/unanet-summarizer/';
+    const SUMMARIZER_STYLESHEET = SUMMARIZER_ROOT + '/summarizer-style.css';
+
     const CONTAINER_ID = 'unanet-summary';
+    const STYLESHEET_ID = 'unanet-style';
```

Comment:
This seems a little too close to normal unanet. `unanet-summarizer-style` maybe?